src/components/vulnerabilities-list.tsx

```diff
@@ -10,10 +10,25 @@ export class VulnerabilitiesList extends React.Component<Props> {
 
     getTableRow(index: number) {
         const {vulnerabilities} = this.props;
+
+        let avdURL: JSX.Element;
```

Comment:
Looking at the if/else statements below the only things that varies is actually `href` attribute and other HTML elements to render the hyperlink are duplicated. So maybe we could just resolve `href` and inline the hyperlink tags in the return statement. WDYT?

```tsx
<Component.TableRow key={vulnID} nowrap>
    <Component.TableCell className="vulnerabilityID">
        <a target="_blank" href={href}>{vulnID}</a
   </Component.TableCell>
</Component.TableRow>
```

Reply:
Nice catch - I updated it to be that here https://github.com/aquasecurity/starboard-lens-extension/pull/27/commits/5458e035a0f89de6e69ea0f5380572f760b8e941